docs/source/instrumentation.rst

```diff
@@ -13,7 +13,8 @@ The use of UDP cleanly isolates Gunicorn from the receiving end of the statsD
 metrics so that instrumentation does not cause Gunicorn to be held up by a slow
 statsD consumer.
 
-To use statsD, just tell Gunicorn where the statsD server is:
+To use statsD, add ``statsd_host = 'localhost:8125'`` in configuration file or
+just tell Gunicorn where the statsD server is:
```

Comment:
Please revert this too. The documentation is already good as is.

You can add a link to http://docs.gunicorn.org/en/stable/settings.html#statsd-host if you want to.